.github/workflows/deploy.yml

```diff
@@ -65,3 +65,16 @@ jobs:
           git push --force-with-lease
         env:
           GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+
+  notifyMattermost:
+    runs-on: ubuntu-latest
+    if: ${{ inputs.environment == 'production' }}
```

Comment:
I would add a check if the deploy step did not fail.

Reply:
@berrydenhartog I added the succes() check, can you check and approve again?